src/components/Create_Group.js

```diff
@@ -1,102 +1,132 @@
-import React, { Component } from 'react'
-import { Link } from 'react-router-dom';
-import { connect } from 'react-redux';
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import { connect } from "react-redux";
 
-import '../style/Create_Group.css';
+import { createPhotosGroup, createPhotosCostume } from "../services/actions";
+import { STORE_TYPES } from "../services/types";
+import { costumes } from '../constants/costumes';
 
-class Create_Group extends Component {
-  constructor(props){
-    super(props);
-    this.state = {
-      groupSelected: undefined,
-      cosSelected: undefined,
-    };
-  }
+import "../style/Create_Group.css";
 
-  // Todo: all state handler in actions
-  selectGroup = (group) => {
-    this.setState({groupSelected: group})
-  }
+const mapStateToProps = state => {
+  return {
+    [STORE_TYPES.STATE.CREATE.GROUP]:
+      state[STORE_TYPES.STATE.CREATE.META][STORE_TYPES.STATE.CREATE.GROUP],
+    [STORE_TYPES.STATE.CREATE.COSTUME]:
+      state[STORE_TYPES.STATE.CREATE.META][STORE_TYPES.STATE.CREATE.COSTUME]
+  };
+};
 
-  handleSelectCos = (event) => {
-    this.setState({cosSelected: event.target.value})
-  }
-  
+const mapDispatchToProps = {
+  createPhotosGroup,
+  createPhotosCostume
+};
+
+class Create_Group extends Component {
```

Comment:
I have a suggestion. We are going to have a lot of `CreateXXX` components. How about lets put them all under `components/create/` folder? 
And also the naming is not consistent with other components like `PhotoList`, `PhotoItem`.
Can we spare the `_`? Just `CreateGroup` should be fine.

Reply:
Ok